ibm/mas_devops/roles/nvidia_gpu/README.md

```diff
@@ -32,9 +32,9 @@ The channel to subscribe to for the gpu operator installation and updates. Avail
 - Default Value: `v1.11`
 
 ### gpu_driver_version
-The gpu driver version image that needs to be pulled from the gpu repository. It is recommended that the right version if GPU driver is used. The MVI installation documentation, the default version below should be used.
+The gpu driver version image that needs to be pulled from the gpu repository. It is recommended that the right version of GPU driver is used. The MVI installation documentation, the default version below should be used.
```

Comment:
This does not read well:
```
The MVI installation documentation, the default version below should be used.

Reply:
I made some corrections. Please check for me. Thank you